apps/react/src/features/anime/components/AnimeSearch/AnimeSearch.tsx

```diff
@@ -0,0 +1,36 @@
+import { getAnimeList } from '@js-camp/react/store/anime/dispatchers';
+import { useAppDispatch } from '@js-camp/react/store/store';
+import { TextField } from '@mui/material';
+
+import React, { FC, memo } from 'react';
+
+import { useEffectSkipFirstRender, useSearch } from '../../../../hooks';
+
+import style from './AnimeSearch.module.css';
+
+export const AnimeSearchInner: FC = () => {
+  const { inputValue, setInputValue, debounceValue } = useSearch();
+  const handleOnChangeValue = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+  ) => {
+    setInputValue(event.target.value);
+  };
+  const dispatch = useAppDispatch();
+  useEffectSkipFirstRender(() => {
+    dispatch(getAnimeList({ search: debounceValue }));
+  }, [debounceValue]);
+
+  return (
+    <div>
+      <TextField
+        value={inputValue}
+        onChange={handleOnChangeValue}
+        className={style['anime-search__input']}
+        placeholder="e.g.Naruto, ..."
```

Comment:
Does the BE supports multiple anime search via separation by comma? If not let's change it to not confuse your users
```suggestion
        placeholder="e.g. Naruto"
```